src/Polly.Net35/CircuitBreaker/ServiceLevelCircuitBreakerState.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using Polly.Utilities;
+
+namespace Polly.CircuitBreaker
+{
+    internal class ServiceLevelCircuitBreakerState : ICircuitBreakerState

```

Comment:
This doesn't really have much to do w/ services. It's just an implementation that uses a success/failure ratio rather than a failure count. A name like `SuccessRatioCircuitBreakerState` might be more appropriate.